### DISCUSSION TITLE
Hot Reload Problem

### DISCUSSION THREAD

**Discussion by InfiniteP** [flet-dev/flet#1881] [Q&A]
Hi, I'm Roni. A while ago, after upgrading Flet to the latest versions, I've been having a problem with Hot Reload; the problem is that it doesn't reload automatically when I make a code change. With previous versions, specifically with the 0.7.x versions I didn't get such an error.

I thought it was the operating system, since I was programming on Windows 10, so I switched to Linux and still the problem persists.

However, the console shows me the following error:

![Error Capture](https://github.com/flet-dev/flet/assets/141675129/23d6c688-7d10-495f-b601-eb58b8d70259)

I don't know if that has anything to do with the problem. Thank you in advance for your help.

Good luck!

**Comment by plugeit** [comment]
Can you send the .py file which you are trying to run and got the above-mentioned error message

**Reply by plugeit** [reply]
and what command you are using to run your app in hot reload in cmd send that too

**Reply by InfiniteP** [reply]
The code executed is a basic template. And the command I execute is "flet run demo.py".

**Comment by bobwatcherx** [comment]
i use flet main.py in flet version new is 0.10.3

but after save my code flet and reload loading . my app is crash and lag

so i must kill and re run flet main.py again . but for version 0.7.2 is good no error

**Reply by InfiniteP** [reply]
That is correct. After such a release the problem started to occur.

**Comment by bobwatcherx** [comment]
^Croot@ubun-HP-Convertible-x360-11-ab1XX:/home/ubun/belajar/fletapk# python3 --version
Python 3.10.12

flet 0.10.3
